### ISSUE TITLE
Reintegrate cloud UI branch

### ISSUE THREAD

**Issue by rjmackay** [ushahidi/platform#813]
- Add feature limits for views
- Add feature limit on activity page
- Add feature limit on data import
- Only include plans page based on a config/build flag?
- Handling for feature limit errors
- Prepopulate config cache from initial config load - this should replace Config service and other custom BootstrapConfig code

**Comment by willdoran** [comment]
This is currently waiting on final review before being merged into develop.
